src/plugins/graphql-operation-logging-plugin.ts

```diff
@@ -9,7 +9,11 @@ import type { Logger } from '@makerx/node-common'
 import { OperationTypeNode } from 'graphql'
 import { omitNil } from '../utils'
 
-export interface GraphQLOperationLoggingPluginOptions<TContext extends GraphQLContext<TLogger, any, any>, TLogger extends Logger = Logger> {
+export interface GraphQLOperationLoggingPluginOptions<
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Perhaps we should just adjust the eslint config to allow any to avoid lots of ignores?